frontend/src/container/MetricsApplication/MetricsPageQueries/MetricsPageQueriesFactory.ts

```diff
@@ -43,14 +44,17 @@ export const getQueryBuilderQueries = ({
 				inputFormat: 'ns',
 				start: store.getState().globalTime.minTime,
 			}),
-			reduceTo: 'sum',
```

Comment:
You can keep this in both cases. It is not relevant for this chart so it doesn't make any difference whether you keep it or remove it.

Reply:
Okay